casper/src/test/scala/coop/rchain/casper/engine/Setup.scala

```diff
@@ -53,18 +52,16 @@ object Setup {
     val spaces = RSpace
       .createWithReplay[Task, Par, BindPattern, ListParWithRandom, TaggedContinuation](store)
       .runSyncUnsafe()
-    val (rspace, replay, historyRepo) = spaces
-    val runtimes =
-      RhoRuntime.createRuntimes[Task](rspace, replay, true, Seq.empty).unsafeRunSync
+    val (rspace, replay) = spaces
+    val historyRepo      = rspace.historyRepo
 
-    val (runtime, replayRuntime) = runtimes
     val (exporter, importer) = {
       (historyRepo.exporter.unsafeRunSync, historyRepo.importer.unsafeRunSync)
     }
     implicit val rspaceStateManager = RSpaceStateManagerImpl(exporter, importer)
 
     implicit val runtimeManager =
-      RuntimeManager.fromRuntimes(runtime, replayRuntime, historyRepo).unsafeRunSync(scheduler)
+      RuntimeManager[Task](rspace, replay, historyRepo).unsafeRunSync(scheduler)
```

Comment:
Are we going to leave it this way? Or the goal to not create manager having initial runtimes passed as input is still on the way?

Reply:
Creating RM from runtimes does not make sense. Existing code is just using runtimes to get RSpace and RSpaceReplay which is exactly what now RM constructor accepts.